pkg/channel/grpc/grpc_channel.go

```diff
@@ -90,7 +92,8 @@ func (g *Channel) invokeMethodV1(ctx context.Context, req *invokev1.InvokeMethod
 	ctx = metadata.NewOutgoingContext(context.Background(), grpcMetadata)
 
 	var header, trailer metadata.MD
-	resp, err := clientV1.OnInvoke(ctx, req.Message(), grpc.Header(&header), grpc.Trailer(&trailer))
+	resp, err := clientV1.OnInvoke(ctx, req.Message(), grpc.Header(&header), grpc.Trailer(&trailer),
+		grpc.MaxCallSendMsgSize(g.maxRequestBodySize*1024*1024), grpc.MaxCallRecvMsgSize(g.maxRequestBodySize*1024*1024))
```

Comment:
I propose to create an opts variable to pass all CallOptions as it seems that in the future these options will only expanding

```suggestion
	var opts []grpc.CallOption
	opts = append(opts, grpc.MaxCallRecvMsgSize(grpc.Header(&header), grpc.Trailer(&trailer), d.maxRequestBodySize*1024*1024), grpc.MaxCallSendMsgSize(d.maxRequestBodySize*1024*1024))
	resp, err := clientV1.OnInvoke(ctx, req.Message(), opts...)
```

Reply:
Done. Pushed the changes.